generator/sources/java/src/com/kaltura/client/KalturaClientBase.java

```diff
@@ -214,57 +220,75 @@ public void queueServiceCall(String service, String action, KalturaParams kparam
 	}
 
 	public void queueServiceCall(String service, String action, KalturaParams kparams, KalturaFiles kfiles, Class<?> expectedClass) {
+		lock();
```

Comment:
locking ReentrantLock should be done in try/finally block.
In addition, there is no following unlock statement